src/network/impl/dialer_impl.cpp

```diff
@@ -7,24 +7,26 @@
 #include <libp2p/connection/stream.hpp>
 #include <libp2p/network/impl/dialer_impl.hpp>
 
+#define TRACE_ENABLED 0
```

Comment:
It is good to have such a way to debug. Despite this, I suggest to remove that define from the production-ready source code.

Reply:
will remove as soon as src is production ready. not now